lib/services/plugins-service.ts

```diff
@@ -178,32 +216,33 @@ export class PluginsService implements IPluginsService {
 	private createPluginsData(pluginsService: ICordovaPluginsService): IFuture<void> {
 		return (() => {
 			var plugins = pluginsService.getAvailablePlugins().wait();
-
 			_.each(plugins, (plugin: Server.CordovaPluginData) => {
-
 				try {
-					var pluginData = pluginsService.createPluginData(plugin).wait();
-				} catch(e) {
+					var data = pluginsService.createPluginData(plugin);
```

Comment:
JavaScript is powerful and the var can be declared in try catch block and used outside it. :smile_cat: I prefer it because it's shorter.